gateway/src/apicast/configuration.lua

```diff
@@ -159,19 +109,7 @@ function _M.parse_service(service)
         app_key = lower(proxy.auth_app_key or 'app_key') -- TODO: use App-Key if location is headers
       },
       rules = map(function(proxy_rule)
-        local querystring_parameters = hash_to_array(proxy_rule.querystring_parameters)
-
-        return {
-          method = proxy_rule.http_method,
-          pattern = proxy_rule.pattern,
-          regexpified_pattern = regexpify(proxy_rule.pattern),
-          parameters = proxy_rule.parameters,
-          querystring_params = function(args)
-            return check_querystring_params(querystring_parameters, args)
-          end,
-          system_name = proxy_rule.metric_system_name or error('missing metric name of rule ' .. inspect(proxy_rule)),
-          delta = proxy_rule.delta
-        }
+        return mapping_rule.from_proxy_rule(proxy_rule)
```

Comment:
This can be further simplified to `map(mapping_rule.from_proxy_rule, proxy.proxy_rules or empty)`

Reply:
Fixed :+1: